src/components/StyleGuide/CardsAsNavigation.mdx

```diff
@@ -0,0 +1,199 @@
+import { Canvas, Meta, Source, Unstyled } from "@storybook/addon-docs/blocks";
+
+import Banner from "../Banner/Banner";
+import Card, { CardActions, CardContent, CardHeading } from "../Card/Card";
+import Link from "../Link/Link";
+import SimpleGrid from "../Grid/SimpleGrid";
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
+import ImageRevamp from "../../utils/ImageExample";
+import { getPlaceholderImage } from "../../utils/utils";
+
+import { distinctLinksCard, fullClickCard } from "./CardsAsNavigationData";
+
+<Meta title="Development Guide/Cards as Navigation Links" />
+
+<ComponentDocsHeader
+  category="Style Guide"
+  componentName="Cards as Navigation Links"
+  summary="Guidelines for for using cards as navigation elements"
+/>
+
+## Table of contents
+
+- [General information](#general-information)
+- [Full-click functionality](#full-click-functionality)
+- [Distinct links](#distinct-links)
+
+## General information
+
+Cards are used extensively throughout NYPL web apps for navigational purposes.
+They are used in search results and on landing pages. They can represent
+distinct items, like books and events, or signify site landing pages and
+sections. They may contain a single link or present multiple links and buttons.
+
+## Full-click functionality
+
+Navigational cards designed to link to a singular destination should be
+configured with [full-click
+functionality](/docs/components-basic-elements-card--docs#card-with-full-click-functionality),
+rendering the entire card as a clickable element.
+
+It is recommended for NYPL web apps to utilize the DS Card component, as it
+provides a hover state for the full-click functionality, thereby indicating that
+the complete card is a clickable element. To review the DS Card styling, please
+consult the <Link href="https://nypl.github.io/design-team-prototypes/full-click-card-styles/#background-image-fade" variant="external">Default background color & image fade</Link> POC.
+
+Currently, for cards not utilizing the DS Card component, identical links are
+frequently applied to both the heading and image within the card. This results
+in a scenario where two distinct elements direct to the same destination.
+Implementing full-click functionality will alleviate this redundancy.
+
+### Examples of full-click cards
```

Comment:
The examples don't seem to really show a hover state except for on the link, is that intentional? If so, may be a little confusing based on the text above.

Reply:
If there's already a plan to update the styles soon, then I think it's ok to keep this as is and it'll be true once the styles are updated. They should be done soon after to reduce any confusion.